docs/SPEC.md

```diff
@@ -457,7 +461,7 @@ Note all numeric types are nullable.
 
 ##### Time types
 
-A _time type_ represents a single point in time with nanosecond precision.
+A _time type_ represents a single point in time with nanosecond precision and a fixed offset from UTC.
```

Comment:
How does time + duration work in the context of a fixed offset? If you add 24h to a time with a fixed offset the day before day light savings you will not correctly honor the time zone since the offset is fixed.

Reply:
I was thinking it wouldn't. I looked at the time encodings and I didn't see anything that encoded time zones and I think the reason is that time zones describe the behavior of how time moves rather than an instant in time. At least that was my thought. We could potentially make it so native addition always uses the option and add a special function for customizing the location. I'll add that.